index.html

```diff
@@ -0,0 +1,103 @@
+    <!-- CSS File -->
+    <link rel="stylesheet" href="styles/index.css" />
+    <title>Youtube Clone with HTML & CSS</title>
+  </head>
+  <body>
+      <header class="header">
+        <div class="logo left">
+            <i id="menu" class="material-icons">menu</i>
+            <img src="./Images/youTubelogo.gif">
+          </div>
+          
+          <div class="search center">
+            <form action="">
+              <input type="text" placeholder="Search" />
+              <button><i class="material-icons">search</i></button>
+            </form>
+            <i class="material-icons mic">mic</i>
+          </div>
+          
+          <div class="icons right">
+            <i class="material-icons">videocam</i>
+            <i class="material-icons">apps</i>
+            <i class="material-icons">notifications</i>
+            <i class="material-icons display-this">account_circle</i>
+          </div>
+      </header>
+      <main>
+          <div class="side-bar">
+            <div class="nav">
+                <a class="nav-link active">
+                  <i class="material-icons">home</i>
+                  <span>Home</span>
+                </a>
+                <a class="nav-link">
+                    <i class="material-icons">local_fire_department</i>
+                    <span>Trending</span>
+                 </a>
+                 <a class="nav-link">
+                    <i class="material-icons">subscriptions</i>
+                    <span>Subscriptions</span>
+                 </a>
+               </div>
+               <hr>
+               <a class="nav-link">
+                <i class="material-icons">library_add_check</i>
+                <span>Library</span>
+             </a>
+             <a class="nav-link">
+                <i class="material-icons">history</i>
+                <span>History</span>
+             </a>
+             <a class="nav-link">
+                <i class="material-icons">play_arrow</i>
+                <span>Your Videos</span>
+             </a>
+             <a class="nav-link">
+                <i class="material-icons">watch_later</i>
+                <span>Watch Later</span>
+             </a>
+             <a class="nav-link">
+                <i class="material-icons">thumb_up</i>
+                <span>Liked Videos</span>
+             </a>
+            </div>
+          </div>
+          <div class="content">
+            <div class="videos">
+                <!-- a video starts -->
+                  <div class="video">
+                     <div class="thumbnail">
+                        <img src="./Images/youtube-thumbnail.jpg" alt="" />
+                      </div>
+              
+                        <div class="details">
+                           <div class="author">
+                              <img src="./Images/genericperson.jpg" alt="" />
+                           </div>
```

Comment:
Make the author's image smaller. This is because it shows up too large when you run the code. Try resizing the image to be much smaller.